src/plugins/http_fetch_operator.py

```diff
@@ -25,56 +31,100 @@ def __init__(
         headers=None,
         http_conn_id="http_default",
         encoding_schema=None,
-        tmp_file=None,
+        tmp_file: Optional[Union[Path, str]] = None,
         output_type=None,
+        xcom_tmp_dir_task_ids: Optional[str] = None,
```

Comment:
Nice addition to use xcom. Although, it does not solve the problem when running Airflow in a distributed fashion. Because the filesystem can be different depending on the node where the executor runs a particular task.

Reply:
Goede. Misschien moeten we een NFS volume, of iets dergelijks, voor inrichten?

Primaire motivatie voor deze toevoeging is dat ik niet wilde dat verschillende DAG runs met 'left overs' van vorige, mogelijk afgebroken' run te maken zouden hebben. Een DAG run specifieke tmp dir voorkomt dat.

Reply:
Wij gebruiken een shared volume voor alle TMP dirs.

XCom is wel leuk, maar nu verliezen we mogelijkheid om bestanden binnen een subfolder te plaatsen.
eg `tmp_file=data/x.json` kan niet gebruikt worden met xcom.

Reply:
I'm not sure I understand you @ngaranko You can't create as deep a folder structure as you like. All this DAG is doing is communicating a tmp dir by means of XCom. Whatever you create within that tmp dir and in whatever folder structure is entire up to the DAG.